test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/BatchTests.cs

```diff
@@ -169,5 +170,131 @@ private void ValidateOperationOutcome(string actualStatusCode, OperationOutcome
             Assert.Equal(expectedIssueType, issue.Code);
             Assert.Equal(expectedDiagnostics, issue.Diagnostics);
         }
+
+        [Theory]
+        [InlineData("true", "true")]
+        [InlineData("true", "false")]
+        [InlineData("false", "true")]
+        [InlineData("false", "false")]
```

Comment:
Why are headerValue and profileValidation needed as separate parameters? It looks like they are only used for the one if statement at the end. This looks like it could have just one of the two parameters and have two theories instead of 4.
Same for the transaction test below.